src/Umbraco.Web.UI/umbraco/config/lang/ru.xml

```diff
@@ -215,6 +215,7 @@
     <key alias="updateDateDesc" version="7.0">Дата/время редактирования документа</key>
     <key alias="updatedBy" version="7.0">Обновлено</key>
     <key alias="uploadClear">Удалить файл</key>
+    <key alias="url">Интернет-ссылка</key>

```

Comment:
This one was removed - have put your translation in the `general_url` key instead :)